src/p2pool.cpp

```diff
@@ -391,6 +391,31 @@ void p2pool::handle_chain_main(ChainMain& data, const char* extra)
 				LOGINFO(0, log::LightCyan() << "You received a payout of " << log::LightGreen() << log::XMRAmount(payout) << log::LightCyan() << " in block " << log::LightGreen() << data.height);
 			}
 			api_update_block_found(&data);
+
+			if(!params().m_onBlockFound.empty())
+			{
+			    std::stringstream cmd;
+				double po = payout;
+				if(po > 0)
+				{
+					po = po * 0.000000000001; //convert to XMR reward
```

Comment:
This will not be needed when you switch to `log::Stream` - just use `log::XMRAmount(payout)`